### PR TITLE
add status to datasets

### PR DESCRIPTION
**Pull request by natevecc**
# Description

Add status to dataset class

## Github Issue
N/A

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Successfully interacted with new version of the API in dev environment 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: blackfynn/models.py [hunk 1]
```diff
@@ -1885,11 +1885,12 @@ def __repr__(self):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 class Dataset(BaseCollection):
-    def __init__(self, name, description=None, **kwargs):
+    def __init__(self, name, description=None, status=None, **kwargs):
```
Comment (rlmv) [inline blackfynn/models.py]: I think you need to also add `status` to the `as_dict` method.

Comment (natevecc) [reply blackfynn/models.py]: Dataset doesn't have a `as_dict` method. Should I add it?

Comment (rlmv) [reply blackfynn/models.py]: It's just hiding right here https://github.com/Blackfynn/blackfynn-python/blob/fbf2366f50f1b729859f118c0eb6184d8f412030/blackfynn/models.py#L2099

Comment (rlmv) [reply blackfynn/models.py]: Gotcha, you should be good then

**Pull request merged**